exercises/hello-world/HINT.md

```diff
@@ -0,0 +1,4 @@
+The starter implementation for this exercise uses [UnsupportedOperationException](http://docs.oracle.com/javase/8/docs/api/?java/lang/UnsupportedOperationException.html). 
+This [exception](https://docs.oracle.com/javase/tutorial/essential/exceptions/) will cause the test to fail with the provided error message. 
```

Comment:
I'd go a little further and explain why we do this: "Throwing exceptions in this manner means the tests will compile but not pass right away." or similar?

Separate point: it seems a little back-to-front to be linking out the the general Exceptions docs after linking to the Javadoc for a specific implementation; is there a way to reorder this hint to avoid that?